<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiPrefsDialogBase</class>
 <widget class="QDialog" name="uiPrefsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>User interface</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Number of entries in a result page</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="pageLenSB">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="collapseDupsCB">
             <property name="toolTip">
              <string>If checked, results with the same content under different names will only be shown once.</string>
             </property>
             <property name="text">
              <string>Hide duplicate results.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabelqtc">
               <property name="text">
                <string>Highlight color for query terms</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="qtermColorLE">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel4">
               <property name="text">
                <string>Result list font</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reslistFontPB">
               <property name="toolTip">
                <string>Opens a dialog to select the result list font</string>
               </property>
               <property name="text">
                <string>Helvetica-10</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetFontPB">
               <property name="toolTip">
                <string>Resets the result list font to the system default</string>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Defines the format for each result list paragraph. Use qt html format and printf-like replacements:&lt;br&gt;%A Abstract&lt;br&gt; %D Date&lt;br&gt; %I Icon image name&lt;br&gt; %K Keywords (if any)&lt;br&gt; %L Preview and Edit links&lt;br&gt; %M Mime type&lt;br&gt; %N Result number&lt;br&gt; %R Relevance percentage&lt;br&gt; %S Size information&lt;br&gt; %T Title&lt;br&gt; %U Url&lt;br&gt;</string>
               </property>
               <property name="text">
                <string>Result paragraph&lt;br&gt;format string</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="rlfTE">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
               <property name="text" stdset="0">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabelHL">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Texts over this size will not be highlighted in preview (too slow).</string>
               </property>
               <property name="text">
                <string>Maximum text size highlighted for preview (megabytes)</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="maxHLTSB">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="useDesktopOpenCB">
             <property name="text">
              <string>Use desktop preferences to choose document editor.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="viewActionPB">
             <property name="text">
              <string>Choose editor applications</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="catgToolBarCB">
             <property name="text">
              <string>Display category filter as toolbar instead of button panel (needs restart).</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoSearchCB">
             <property name="text">
              <string>Auto-start simple search on whitespace entry.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="initStartAdvCB">
             <property name="text">
              <string>Start with advanced search dialog open.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="keepSortCB">
             <property name="text">
              <string>Remember sort activation state.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="previewHtmlCB">
             <property name="text">
              <string>Prefer Html to plain text for preview.</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>Search parameters</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1_2">
               <property name="text">
                <string>Stemming language</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="stemLangCMB"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="autoPhraseCB">
             <property name="toolTip">
              <string>A search for [rolling stones] (2 terms) will be changed to [rolling or stones or (rolling phrase 2 stones)]. 
This should give higher precedence to the results where the search terms appear exactly as entered.</string>
             </property>
             <property name="text">
              <string>Automatically add phrase to simple searches</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line2">
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="buildAbsCB">
             <property name="toolTip">
              <string>Do we try to build abstracts for result list entries by using the context of query terms ? 
May be slow for big documents.</string>
             </property>
             <property name="text">
              <string>Dynamically build abstracts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="replAbsCB">
             <property name="toolTip">
              <string>Do we synthetize an abstract even if the document seemed to have one?</string>
             </property>
             <property name="text">
              <string>Replace abstracts from documents</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Synthetic abstract size (characters)</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="syntlenSB">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>80</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>250</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Synthetic abstract context words</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="syntctxSB">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>20000</number>
               </property>
               <property name="value">
                <number>4</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line3">
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel14">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>The words in the list will be automatically turned to ext:xxx clauses in the query language entry.</string>
               </property>
               <property name="text">
                <string>Query language magic file name suffixes.</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="autoSuffsCB">
               <property name="text">
                <string>Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="autoSuffsLE">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="spacer2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>70</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ExtraDb">
        <attribute name="title">
         <string>External Indexes</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QListWidget" name="idxLV">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QPushButton" name="togExtraDbPB">
               <property name="text">
                <string>Toggle selected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="actAllExtraDbPB">
               <property name="text">
                <string>Activate All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unacAllExtraDbPB">
               <property name="text">
                <string>Deactivate All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line2_2">
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QPushButton" name="delExtraDbPB">
               <property name="toolTip">
                <string>Remove from list. This has no effect on the disk index.</string>
               </property>
               <property name="text">
                <string>Remove selected</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacer3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addExtraDbPB">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>Click to add another index directory to the list</string>
               </property>
               <property name="text">
                <string>Add index</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <spacer name="Horizontal Spacing2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>210</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk">
         <property name="toolTip">
          <string>Apply changes</string>
         </property>
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="toolTip">
          <string>Discard changes</string>
         </property>
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
